netket/nn/linear.py

```diff
@@ -314,6 +318,95 @@ def __call__(self, inputs: Array) -> Array:
         return y
 
 
+class DenseEquivariant(Module):
+    """An equivariant layer that takes a feature map with embeddings corresponding to various
+    poses of a symmetry group, shape = [n_symm*in_features] and outputs a feature map of identical
```

Comment:
maybe i am ignorant, but what is a pose of a symmetry group?

Reply:
Pose is an ML term, is the word orientation more clear? It's quite tricky to explain what is going on in a paragraph, that's why I plan to write more stuff

Reply:
I can only guess what you mean by orientation...

As a short description I would specify carefully what should be the inputs and outputs and with maybe easy/to/understand words what it means. 
Then newline, 
then try to explain concepts you need to say what this does. 
Links to websites/papers are welcome 
you write a link with the syntax 
```python
` text <https:://link.to.something>`_
```